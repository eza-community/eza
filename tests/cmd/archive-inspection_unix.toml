bin.name = "eza"
args = "tests/itest --inspect-archives --recurse"
