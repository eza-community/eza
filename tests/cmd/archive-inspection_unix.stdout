a
b
c
complex_tar.tar
d
dir-symlink -> vagrant/debug
e
exa
f
g
h
i
image.jpg.img.c.rs.log.png
index.svg
j
k
l
m
n
not_a_tar.tar
o
p
q
recursive_tar.tar
simple_tar.tar
vagrant

tests/itest/complex_tar.tar:
a
b
c
dir-symlink -> vagrant/debug
vagrant

tests/itest/complex_tar.tar/vagrant:
vagrant/debug
vagrant/dev
vagrant/log

tests/itest/complex_tar.tar/vagrant/debug:
vagrant/debug/.gitignore
vagrant/debug/a
vagrant/debug/symlink -> a
vagrant/debug/symlink-broken -> ./b

tests/itest/complex_tar.tar/vagrant/dev:
vagrant/dev/main.bf

tests/itest/complex_tar.tar/vagrant/log:
vagrant/log/file.png
vagrant/log/run

tests/itest/complex_tar.tar/vagrant/log/run:
vagrant/log/run/run.log.text
vagrant/log/run/sps.log.text

tests/itest/not_a_tar.tar:

tests/itest/recursive_tar.tar:
completions
man
TESTING.md

tests/itest/recursive_tar.tar/completions:
completions/bash
completions/fish
completions/nush
completions/zsh

tests/itest/recursive_tar.tar/completions/bash:
completions/bash/eza

tests/itest/recursive_tar.tar/completions/fish:
completions/fish/eza.fish

tests/itest/recursive_tar.tar/completions/nush:
completions/nush/eza.nu

tests/itest/recursive_tar.tar/completions/zsh:
completions/zsh/_eza

tests/itest/recursive_tar.tar/man:
man/eza.1.md
man/eza_colors-explanation.5.md
man/eza_colors.5.md

tests/itest/simple_tar.tar:
CONTRIBUTING.md
README.md
TESTING.md

tests/itest/exa:
file.c -> djihisudjuhfius
sssssssssssssssssssssssssggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggsssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss

tests/itest/exa/sssssssssssssssssssssssssggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggsssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss:
Makefile

tests/itest/vagrant:
debug
dev
log

tests/itest/vagrant/debug:
a
symlink -> a
symlink-broken -> ./b

tests/itest/vagrant/dev:
main.bf

tests/itest/vagrant/log:
file.png
run

tests/itest/vagrant/log/run:
run.log.text
sps.log.text
